src/Microsoft.AspNetCore.OData/Extensions/HttpRequestExtensions.cs

```diff
@@ -26,6 +26,8 @@ namespace Microsoft.AspNetCore.OData.Extensions
     /// </summary>
     public static class HttpRequestExtensions
     {
+        private static readonly string ODataInstanceAnnotationContainerKey = "odataInstanceAnnotation_14802D58-69EF-4B28-9BDC-963D3648F06A";
```

Comment:
Would `const` be better here since the value is known at compile-time? (I assume we'll never change this)
```suggestion
        private const string ODataInstanceAnnotationContainerKey = "odataInstanceAnnotation_14802D58-69EF-4B28-9BDC-963D3648F06A";
```